HT-2.py

```diff
@@ -0,0 +1,67 @@
+# Підраховує кількість голосних літер у тексті
```

Comment:
Still present linter errors. Better try to fix them
(venv) [antoha@dlhn test_project]$ ./bin/lint ./T-HOMETASKS/L02/l02_h01_Margarita.py
./T-HOMETASKS/L02/l02_h01_Margarita.py:1:1: D100 Missing docstring in public module
./T-HOMETASKS/L02/l02_h01_Margarita.py:4:1: D103 Missing docstring in public function
./T-HOMETASKS/L02/l02_h01_Margarita.py:18:1: D103 Missing docstring in public function
./T-HOMETASKS/L02/l02_h01_Margarita.py:19:11: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/l02_h01_Margarita.py:21:11: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/l02_h01_Margarita.py:23:15: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/l02_h01_Margarita.py:24:11: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/l02_h01_Margarita.py:46:1: D103 Missing docstring in public function
./T-HOMETASKS/L02/l02_h01_Margarita.py:52:27: C812 missing trailing comma
./T-HOMETASKS/L02/l02_h01_Margarita.py:55:21: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/l02_h01_Margarita.py:66:26: W292 no newline at end of file